sql/pg_mooncake--0.1.1--0.1.2.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE mooncake.delta_update_records (
+    -- Postgres transaction id, which inserts the record into the table.
+    txn_id INTEGER,
+    -- Timestamp when the update record is made.
+    ts TIMESTAMPTZ NOT NULL DEFAULT now(),
+    -- Table OID which the update record is made for.
+    oid OID,
+    -- Directory, where delta table and parquet files are stored.
+    dir TEXT,
+    -- Json format for delta table options.
+    delta_option TEXT,
```

Comment:
We can remove columns `delta_option`, `dir`, `txn_id`. The first two can be derived using relation oid. `txn_id` can be retrieved directly by tuple `xmin`.

nit: file_paths -> file_names